ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
1. Can we use `pkg` in `basePath`?
2. Need to manually add old versions (those not existed in CDNJS)or think about a better pattern.
cc @sufuf3